### ISSUE TITLE
BetterTTV doesn't show twitch chat and fails to load pages

### ISSUE THREAD

**Issue by yeongy** [night/BetterTTV#1903]
Whenever I have the BetterTTV extension enabled along with FrankerFaceZ, it causes twitch chat to now show anyone chatting and causes twitch pages to not load. 
![amw64yv](https://cloud.githubusercontent.com/assets/24642644/21299174/1a398a3a-c567-11e6-8e3a-fd78e06a2284.png)
![gfcgl7i](https://cloud.githubusercontent.com/assets/24642644/21299178/1f24b754-c567-11e6-9069-cdc90ade0701.png)

Even with all other chrome extensions off this problem occurs

**Comment by night** [comment]
If you check that error log, there's a fatal Ember error within FrankerFaceZ. Maybe @SirStendec can take a look, since the error doesn't bubble up to our tracking.

**Comment by SirStendec** [comment]
Alright, what I think is happening here is that FrankerFaceZ is trying to re-position the player at the same time that BetterTTV is telling the player's owner view to rerender and Ember doesn't like when things cause views to re-render while they're already updating or rendering.

I've deployed an update to FrankerFaceZ that checks if BetterTTV is present and, if so, it won't try re-positioning the player. This honestly should've been in before, but when I updated for the mini-player, BetterTTV hadn't yet had its own update.

Regarding the error `TypeError: Cannot read property 'getBadges' of undefined`, this is probably caused by Ember breaking from the aforementioned rendering issue. I did notice a related potential bug, but I doubt it's an issue as it only comes up after a catastrophic Ember error in my testing. (In [line 553 of src/js/chat/helpers.js](https://github.com/night/BetterTTV/blob/5540494acbedbe4af18b5e712dfb58c0b3ad783d/src/js/chat/helpers.js#L553), BetterTTV seems to be checking if a function exists but calls it within the conditional.)
